exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,8 @@
 // Declare your function first
-
+function name(a,b) {
+    return a/b
```

Comment:
What would happen if `b = 0`; you would execute `a/0` - give this a try and see what happens; have a think about how you might handle that situation (note - this is more advanced than strictly necessary at this point, but it's always good to think ahead)